### PR TITLE
Release 2.5.0

### PR DESCRIPTION
**Pull request by miguel-chacon**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Description of the Change
New release 2.5.0
See CHANGELOG

### Applicable Issues
#109 #139

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,8 +4,7 @@ LABEL maintainer="labs@idealista.com"
 
 EXPOSE 8089
 
-RUN apk add gcc libc-dev linux-headers --update-cache \
-    && rm -rf /var/cache/apk/*
+RUN apk add gcc libc-dev linux-headers --no-cache
```
Comment (dortegau) [inline Dockerfile]: shouldn't cache be updated?

File: README.md [hunk 2]
```diff
@@ -68,6 +68,54 @@ Prom2teams uses Flask and, to have the service monitored, we use @rycus66's [Pro
...
+
+#### Installing the Chart
+
+To install the chart with the release name `my-release` run:
+
+```bash
+$ helm install --name my-release /location/of/prom2teams_ROOT/helm
+```
+
+After a few seconds, Prom2Teams should be running.
```
Comment (dortegau) [inline README.md]: Not very important, but we are using `prom2teams` in lowercase :)

File: Dockerfile (hunk 1 above)
Comment (miguel-chacon) [reply Dockerfile]: with --no-cache flag always do a fetch before because it isn't use any local cache at all.

File: README.md (hunk 2 above)
Comment (miguel-chacon) [reply README.md]: :+1: fixed

**Pull request merged**